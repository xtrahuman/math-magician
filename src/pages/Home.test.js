import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Home from './Home';

it('renders correctly', () => {
  const home = renderer
    .create(
        <Home/>,
      )
    .toJSON();
  expect(home).toMatchSnapshot();
});

