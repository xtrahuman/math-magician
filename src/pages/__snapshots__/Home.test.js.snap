// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="math-home"
>
  <h2>
    Welcome to our page
  </h2>
  <p>
    Given that math content is immersed in a linguistic medium, we wondered whether one of the languages of bilinguals is a preferred vehicle for math and, importantly, whether this linguistic context always remains on the surface or at some point merges with numerical representation.
  </p>
  <p>
    Geometry is initially the study of spatial figures like circles and cubes, though it has been generalized considerably. Topology developed from geometry; it looks at those properties that do not change even when the figures are deformed by stretching and bending, like dimension.
  </p>
  <p>
    A differential equation is an equation involving an unknown function and its derivatives. In a dynamical system, a fixed rule describes the time dependence of a point in a geometrical space. The mathematical models used to describe the swinging of a clock pendulum, the flow of water in a pipe, or the number of fish each spring in a lake are examples of dynamical systems.
  </p>
</div>
`;
